homeassistant/components/yolink/sensor.py

```diff
@@ -141,7 +146,8 @@ def cvt_volume(val: int | None) -> str | None:
         device_class=SensorDeviceClass.HUMIDITY,
         native_unit_of_measurement=PERCENTAGE,
         state_class=SensorStateClass.MEASUREMENT,
-        exists_fn=lambda device: device.device_type in [ATTR_DEVICE_TH_SENSOR],
+        exists_fn=lambda device: device.device_type in [ATTR_DEVICE_TH_SENSOR]
```

Comment:
Please use a parenthesis around the lambda and indent it to make it easier to distinguish it from the entity parameters when the lambda spans more than one line.